src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = "CORECLR";
+        private static Architecture? s_processArch = null;
+        private static bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string GetRuntimeDebugName()
+        {
+            return RuntimeName;
```

Comment:
@Priya91 Just so we're on the same page. Those should be pure IL and not ship with .NET Framework itself right? We wouldn't type forward such a simple type.

Also I think it should be:

".NET Framework {version}"
".NET Core {version}"
".NET Native {version}"